pkg/api/types.go

```diff
@@ -1873,3 +1873,11 @@ type ComponentStatusList struct {
 
 	Items []ComponentStatus `json:"items"`
 }
+
+type ServiceIPAllocation struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty"`
+
+	CIDR string `json:"cidr"`
```

Comment:
I think I might want to make this object more generic so I can reuse it for other allocations of integer ranges and not add one type per (UID and port allocation can both use this).